docs/commands_v2.rst

```diff
@@ -0,0 +1,313 @@
+
+Then from the bash command line::
+
+    $ export KADI_SCENARIO=nsm-cti
+    $ export PYTHONPATH=$HOME/git/kadi:$HOME/git/parse_cm  # for Ska3 2022.2
+    $ dpa_check \
+        --outdir=out-cti \
+        --oflsdir=DAWG-demo/OCT2521/oflsb \
+        --state-builder=sql \
+        --run-start=2021:296:18:00:00
+
+Configuration options
+---------------------
+
+The kadi commands configuration options are stored in the file
+``~/.kadi/config/kadi.cfg``. The location of this file is fixed.
+
+The available options with the default settings are as follows::
+
+    [commands]
+    ## Default lookback for previous approved loads (days).
+    default_lookback = 30
+
+    ## Cache backstop downloads in the astropy cache. Should typically be False,
+    ## but useful during development to avoid re-downloading backstops.
+    cache_loads_in_astropy_cache = False
+
+    ## Clean backstop loads (like APR1421B.pkl.gz) in the loads directory that are
+    ## older than the default lookback. Most users will want this to be True, but
+    ## for development or if you always want a copy of the loads set to False.
+    clean_loads_dir = True
+
+    ## Directory where command loads and command events are stored after
+    ## downloading from Google Sheets and OCCweb.
+    commands_dir = ~/.kadi
+
+    ## Default version of kadi commands ("1" or "2").  Overridden by
+    ## KADI_COMMANDS_VERSION environment variable.
+    commands_version = 1
+
+    ## Google Sheet ID for command events (flight scenario).
+    cmd_events_flight_id = 19d6XqBhWoFjC-z1lS1nM6wLE_zjr4GYB1lOvrEGCbKQ
+
+    ## Half-width box size of star ID match for get_starcats() (arcsec).
+    star_id_match_halfwidth = 5
+
+    ## Half-width box size of fid ID match for get_starcats() (arcsec).
+    fid_id_match_halfwidth = 40
+
+Modify options
+^^^^^^^^^^^^^^
+
+To modify a configuration there a few options. First is programmatically within
+Python to change a parameter for all subsequent code::
+
+    >>> from kadi.commands import conf, get_cmds
+    >>> conf.default_lookback
+    30
+    >>> conf.default_lookback = 60
+
+You can also temporarily change an option within a context manager::
+
+    >>> with conf.set_temp('commands_version', '2'):
+    ...     cmds2 = get_cmds('2022:001', '2022:002')  # Use commands v2
+    >>> cmds1 = get_cmds('2022:001', '2022:002')  # Use commands v1
+
+For an even-more permanent solution you can write out the configuration file
+to disk and then edit it. This could be a good option if you want to always
+use commands version v2 for testing purposes.
+
+    >>> import kadi
+    >>> status = kadi.create_config_file()
+    INFO: The configuration file has been successfully written to
+    ~/.kadi/config/kadi.cfg [astropy.config.configuration]
+
+
+Environment variables
+---------------------
+
+``KADI``
+  Override the default location of kadi flight data files ``cmds2.h5`` and
+  ``cmds.pkl``.
```

Comment:
I assume this is still cmds2.pkl in this context, but that's a nit.

Reply:
Fixed.

Reply:
Fixed.